addons/web/static/src/public/utils.js

```diff
@@ -109,3 +109,53 @@ export function makeButtonHandler(fct) {
         return result;
     };
 }
+
+/**
+ * Patches a "t-" entry of a dynamic content.
+ *
+ * @param {Object} dynamicContent
+ * @param {string} selector
+ * @param {string} t
+ * @param {any|function} replacement, if a function, takes the element and the
+ *     replaced's function output as parameters
+ */
+export function patchDynamicContentEntry(dynamicContent, selector, t, replacement) {
```

Comment:
It would be nice to have a complete docstring example here I think :) (something kind of like the test)